packages/cypress/internal/install-cypress.js

```diff
@@ -10,30 +10,36 @@
  */
 
 const {spawnSync, spawn} = require('child_process');
-const {readdirSync, statSync, writeFileSync, mkdirSync} = require('fs');
+const {readdirSync, statSync, writeFileSync, mkdirSync, createWriteStream} = require('fs');
 const {
   join,
   basename,
   relative,
   dirname,
 } = require('path');
+
 const nodePath = process.argv[1];
 const cwd = process.cwd();
 const cypressBin = process.argv[2];
 
-// Sandboxing doesn't work on windows, so we can just use the global cypress cache.
-if (process.platform === 'win32') {
-  installGlobalCypressCache();
-  process.exit(0);
-}
+const tarPath = join(cypressBin.split('node_modules')[0], 'node_modules', 'tar', 'index.js')
```

Comment:
are you mutating the node_modules tree? seems like it would be better to put that somewhere else

Reply:
Resolved